docs/contributors/github-labels.md

```diff
@@ -34,6 +34,7 @@ The labels in this list originated within Kubernetes at
 | area/flow-visibility/elk           | Issues or PRs related to the reference ELK configuration for flow visualization | Any |
 | area/flow-visibility/export        | Issues or PRs related to the Flow Exporter functions in the Agent | Any |
 | area/github-membership             | Categorizes an issue as a membership request to join the antrea-io Github organization | Any |
+| area/grouping                      | Issues or PRs related to ClusterGroup, Group API | Any |
```

Comment:
would this apply to anything besides network policies?

Reply:
Finally we want to support applying more policies to a group. But I am ok if we put grouping under network policy for now.

Reply:
i was thinking `Egress` or other resources could also support them; that was one of the main reasons why we refactored the grouping controller and make it independent of network policy

Reply:
but we could also update it later when we actually use it outside of NP